src/plugins/openInNewTabAction/openInNewTabAction.js

```diff
@@ -0,0 +1,58 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2021, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+export default class OpenInNewTab {
+    constructor(openmct) {
+        this.name = 'Open In New Tab';
+        this.key = 'newTab';
+        this.description = 'Open in a new browser tab';
+        this.group = "windowing";
+        this.priority = 10;
+        this.cssClass = "icon-new-window";
+
+        this._openmct = openmct;
+    }
+    invoke(objectPath) {
+        let url;
+        let urlParams = this._openmct.router.getParams();
+        // Needs to include start/end bounds timeSystem and mode in new URL
+        // to keep time consistent
+        let newTabParams = [];
```

Comment:
Let's externalize any URL handling, like @mariuszr has [in their submission](https://github.com/nasa/openmct/pull/3866/files#diff-135f986fa11d30188fa6ce3f8e724fd4a87e5c56533faf0798db3b8634758b5fR26).

We are not really using the service pattern any more, so we could just name it `url.js` and put it under the `/src/tools' folder and include it here.

The new util function could work something like this:
```JavaScript
export function objectPathToUrl(objectPath, params){
//objectPath is an array of objects
//`params` is a params object, ie. key-value pairs
}
```
This util function should also be used from [object-link.js](https://github.com/nasa/openmct/blob/master/src/ui/mixins/object-link.js#L21), which is doing the same thing (but neglecting parameters right now).

Reply:
When making this change make sure to recognize @mariuszr using a [`Co-authored-by` trailer in the commit message](https://docs.github.com/en/github/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors).

Reply:
@akhenry I've made the changed accordingly :)